README.md

```diff
@@ -12,26 +12,34 @@ for other blockchain protocols implementations.
 
 ## Getting started
 
-### Prerequisites
-
 If it's the first time you boot a `nodeos` node, please review
 https://developers.eos.io/welcome/latest/tutorials/bios-boot-sequence
 and make sure you get a grasp of what this blockchain node is capable.
 
-You might want to have the [eosio.cdt tools](https://github.com/EOSIO/eosio.cdt)
```

Comment:
I would like to add a quick sentence here:

```
Don't be intimidated by the `nodeos` binary and how to correctly start it. The dfuse for EOSIO single binary is going to start and operate the `nodeos` process for you, abstracting this complexity away. 

Simply ensure you have a deep mind instrumented `nodeos` process on your machine by following the [Requirements](#requirements) section than heads-on to [Installing](#installing) section for installation instructions.
```

Edits welcome, my goal is to emphasis that you don't need to start `nodeos` youself.